src/app/components/views/home/home.component.html

```diff
@@ -1 +1,18 @@
-<p>home works!</p>
+
+<div>
+    <div class="text1">
+        <h3 class="heading">Store your important urls in a smart way</h3>
+        <p class="para">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur voluptas totam quibusdam nihil! Odio distinctio cum in magni enim facere quaerat minus itaque! Ullam enim obcaecati similique laboriosam tempore beatae.</p>
+    </div>
+    
+    <img class="designer" alt="Step Three Image" src="assets/designer.png" >
```

Comment:
Have more descriptive alt text.[https://supercooldesign.co.uk/blog/how-to-write-good-alt-text]( https://supercooldesign.co.uk/blog/how-to-write-good-alt-text)

Reply:
done